tests/fixtures/logs-composefile/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+simple:
+  image: busybox:latest
+  command: sh -c "echo hello && sleep 5 && echo world"
+another:
+  image: busybox:latest
+  command: sh -c "sleep 6 && echo test"
```

Comment:
These sleeps make the tests excessively slow. It also makes them potentially flaky if running on a heavily loaded server.

I believe the `sleep` is used to test follow vs no-follow behaviour.
What do you think about doing this instead:
* have `simple` service `sleep 200` , and `another` service without sleep
* when testing follow you can assert that the process is still running after getting the first log line out. Then `proc.terminate()` (or `os.kill()`) the logs command so we don't wait on the follow to finish. You can do this by calling `start_process()` instead of dispatch (there are a few examples of this).

Testing no-follow would be the same.
Testing other flags should never use `-f` so that we don't wait for logs to finish.